src/backend/joanie/tests/core/test_api_admin_contract_definitions.py

```diff
@@ -148,6 +148,43 @@ def test_admin_api_contract_definition_list_filter_by_language_invalid(self):
             },
         )
 
+    def test_admin_api_contract_definition_list_filter_by_id(self):
+        """
+        Staff user should be able to get a paginated list of contract definition
+        filter by id.
```

Comment:
```suggestion
        Staff user should be able to get a list of contract definition filter by id.
```
I would suggest to test the key "next" from the list in return to see the paginated elements. Or just take away the word "paginated". Maybe I understand the term wrong.

What would happen if you parse an Id that does not exist, and if the id is not uuid ?